hack/update-version-list.sh

```diff
@@ -35,9 +35,9 @@ function updateVersionsInIstioTypeComment() {
       -e "s/(\+kubebuilder:default=.*version: \")[^\"]*\"/\1$defaultVersion\"/g" \
       api/v1/istio_types.go api/v1/istiocni_types.go
     
-    cniSelectValues=$(yq '.versions[]| select(. | has("ref") | not) | .name | ", \"urn:alm:descriptor:com.tectonic.ui:select:" + . + "\""' "${VERSIONS_YAML_PATH}" | tr -d '\n')
-    cniVersionsEnum=$(yq '.versions[]| select(. | has("ref") | not) | .name' "${VERSIONS_YAML_PATH}" | tr '\n' ';' | sed 's/;$//g')
-    cniVersions=$(yq '.versions[]| select(. | has("ref") | not) | .name' "${VERSIONS_YAML_PATH}" | tr '\n' ',' | sed -e 's/,/, /g' -e 's/, $//g')
+    cniSelectValues=$(yq '.versions[] | select (.eol != true) | select(.ref == null) | .name | ", \"urn:alm:descriptor:com.tectonic.ui:select:" + . + "\""' "${VERSIONS_YAML_PATH}" | tr -d '\n')
```

Comment:
CNI can't use aliases?

Reply:
hmm. yeah, strange, I didn't introduce this, but it seems we're not adding the aliases to the dropdown, but it is possible

Reply:
Sorry, I'm blind. These values are for `istiorevision_types.go`. Values for `istiocni_types.go` correctly contain the aliases.